wordpress-mobile/WordPress-iOS
# Fix horizontal scrolling in plugin directory

Fixes #16087. The rows in the plugins directory weren't scrollable, and tapping on items didn't display the correct details for an individual item. I've fixed this by adding the collectionview to the cell's content view instead of the main view itself.

![plugins-1](https://user-images.githubusercontent.com/4780/118849428-a4ee1080-b8c7-11eb-9fd7-c2a256d4e2fd.gif)

**To test**

* Build and run
* Navigate to a site with plugins (e.g. a Jetpack site)
* Go to Plugins, and ensure you can scroll each row
* Check in landscape orientation too
* Tap on items in the row and ensure the detail for that plugin is displayed

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
